src/Data/Connection/PostObjectConnectionResolver.php

```diff
@@ -601,11 +601,12 @@ public function get_args(): array {
 		 * Filters the GraphQL args before they are used in get_query_args().
 		 *
 		 * @param array                        $args                The GraphQL args passed to the resolver.
+		 * @param array                        $original_args       Unsanitized args passed to the resolver.
 		 * @param PostObjectConnectionResolver $connection_resolver Instance of the ConnectionResolver
 		 *
 		 * @since 1.11.0
 		 */
-		return apply_filters( 'graphql_post_object_connection_args', $args, $this );
+		return apply_filters( 'graphql_post_object_connection_args', $args, $this->args, $this );
```

Comment:
The filter should be able to access the args from the 2nd argument already, eh?

Reply:
Also, this would be a breaking change. We'd need to add new args after the existing args.

Reply:
Nope because the `$args` is a protected parameter.

Reply:
ah, and calling `$this->get_args()` is too late because of [this](https://github.com/wp-graphql/wp-graphql/blob/develop/src/Data/Connection/AbstractConnectionResolver.php#L174), eh?

Reply:
starting to remember what we went through last week 😆

Reply:
ok, ya, I think we could pass `$this->args, $this->context, $this->info` like we do in this filter: https://github.com/wp-graphql/wp-graphql/blob/develop/src/Data/Connection/AbstractConnectionResolver.php#L445


We need to add new args after the existing args though, otherwise it breaks existing code using the filter

Reply:
I'll update the code then. One sec.

Reply:
Done.